README.md

```diff
@@ -47,13 +47,14 @@ You can find a demo of the project [here](https://github.com/reubenae/user-data-
 ## Contributing
 
 1. Fork the repo
-1. Install dependencies locally `yarn` or `npm install`
-1. Make your changes.
-1. Make sure it builds `yarn build` or `npm build`
-1. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
-1. Test your changes in your consuming code: `yarn link` [Yarn link documentation](https://classic.yarnpkg.com/en/docs/cli/link) or `npm link` [npm link documentation](https://docs.npmjs.com/cli/link)
-1. Ensure the code coverage is the same or higher than before your changes.
-1. Create a PR to the `master` branch
+2. Install dependencies locally `yarn` or `npm install`
+3. Make your changes.
+4. Make sure it builds `yarn build` or `npm build`
+5. Run the tests (you added tests, right?) with `npm test` or `yarn tests`
```

Comment:
Please revert this change. Numbered lists in markdown are supposed to be all 1's - when the markdown is processed they get the correct numbers added. It means you can insert an extra step earlier on without having to modify all the subsequent steps.

Rendered view in master: 
![image](https://user-images.githubusercontent.com/17691502/94835356-7dfecf00-0409-11eb-991d-f0670a68e3f0.png)